www/index.html

```diff
@@ -440,7 +440,9 @@ <h3 id="other" data-i18n="about-other-platforms">Other platforms/versions</h3>
                     <br/>
                     <br/>
                 </div>
-
+                <div id='library' style="display: none; color: white;">
+                    <iframe width="100%" id="libraryContent" src="/www/library.html"></iframe>
```

Comment:
I think the problem is that you're loading this with an absolute URL (starting with `/www`), but we can't always be sure what that is going to serve. You should use a URL that is relative to the document loading the iframe, i.e. use `library.html`, and test that this works. This is probably the issue. I'm not serving from the same base as you are. I'm serving from my `Repos` directory, not from inside the `kiwix-js` directory. So, the file can't be found at `Repos/www/library.html`. Always use fully relative references, because you can't control where the base directory is in relation to your content!